chart/demoscenario.sh

```diff
@@ -0,0 +1,96 @@
+set -o errexit
+
+NAME=$1
```

Comment:
Would be great to have some sort of parameters verification here, let's log a follow-up issue since this script will change a lot in the future.

Also, help prompt would be great.

Reply:
We will add it to our backlog.

Reply:
We'll add your suggestion to the backlog and implement it in the future when it'll grow. For now this script shows the opportunities, rather than does anything reasonable, so I think it might remain in this form.